.gitlab/junit_upload/junit_upload.yml

```diff
@@ -0,0 +1,54 @@
+---
+unit_tests_arm64_windows_junit_upload:
+  stage: junit_upload
+  rules:
+    - !reference [.except_mergequeue]
+    - when: always
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  needs:
+    - tests_deb-arm64-py3
+    - tests_rpm-arm64-py3
+    - tests_windows-x64
+    - tests_windows-x64-fast-v2
+    - integration_tests_windows-x64
+  script:
+    - source /root/.bashrc
+    # Upload generated junit files
+    - export DATADOG_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.datadog_api_key_org2)
+    - for f in junit-*.tgz; do inv -e junit-upload --tgz-path $f; done
+
+system_probe_arm64_junit_upload:
+  stage: junit_upload
+  rules:
+    - !reference [.except_mergequeue]
+    - when: always
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  allow_failure: true
+  needs:
+    - job: kernel_matrix_testing_run_system_probe_tests_arm64
+      parallel:
```

Comment:
Do you need the `parallel` for a needs? We didn't have it before.

Reply:
I think so (I think I faced an issue but I'm not sure anymore) 
I'll try your solution to get rid of this job, so the question vanishes